lib/contentful_model/manageable.rb

```diff
@@ -0,0 +1,104 @@
+module ContentfulModel
+  module Manageable
+    attr_reader :dirty
+
+    def initialize(*args)
+      @dirty = false
+      define_setters
+    end
+
+    def to_management
+      update_management_entry_fields(management_entry)
+    end
+
+    def save
+      result = to_management.save
+      @dirty = false
+      self
+    end
+
+    def publish
+      to_management.publish
+      self
+    end
+
+    protected
+
+    def update_management_entry_fields(management_entry)
+      map_management_fields(management_entry, fields)
+    end
+
+    private
+
+    def management_space
+      @management_space ||= self.class.management(default_locale: locale).spaces.find(space.id)
+    end
+
+    def management_entry
+      # Fetches always in case of Version changes to avoid Validation errors
```

Comment:
This feels dangerous to me if we consider it in conjunction with how `to_management` is implemented. If I understand it correctly, the local field values always win, even when they were never modified locally. I think we should have a more sophisticated merging of remote and local entries.